OKTA-501360 Update title and add nut-facts to scim-provisioning-integration-conne… (#3507)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

meta:
  - name: description
    content: Use Okta to make SCIM requests to your SCIM API service by creating and configuring an Okta SCIM integration from the Admin Console.
layout: Guides
---

This guide teaches you how to configure Okta to make SCIM requests to your SCIM API service.

---

**Learning outcomes**

* Create an Okta SCIM integration.
* Configure SCIM provisioning.
* Verify user profile and attribute mappings.

**What you need**

* A SCIM API service
* An [Okta Developer Edition org](https://developer.okta.com/signup/)

---

## Getting connected

Make sure your SCIM implementation passes all the Runscope tests before integrating it with Okta. For more information on testing your SCIM implementation, see [Prepare your SCIM API service](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api)

Begin by signing in to your Okta Developer Edition org. If you don't have one, sign up for an [Okta developer account](https://developer.okta.com/signup/).

1. Sign in to your [Okta org](https://developer.okta.com/login/) with your administrator account.

> 1. Click the **Admin** button on the top right of the page.

Reviewer: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/upper-left-upper-right

```suggestion
1. Click **Admin** in the upper-right right corner of the page.
```
